ext/extralite/database.c

```diff
@@ -386,6 +386,59 @@ VALUE Database_prepare(VALUE self, VALUE sql) {
   return rb_funcall(cPreparedStatement, ID_NEW, 2, self, sql);
 }
 
+/* call-seq:
+ *   Database.status(op, need_watermark, reset)
+ *
+ * Returns sqlite3 global status in the form of op_value or [ op_value, op_highwatermark ]
+ * or throws an exception. All arguments are optional.
+ */
+VALUE Database_singleton_status(int argc, VALUE* argv, VALUE self) {
```

Comment:
I haven't look to deep into the [docs](https://sqlite.org/c3ref/status.html), but I don't see why there's a need to supply the method with a `wmrk` argument. It would be simpler to do `status(op, reset_flag) => [current, high_water]`.

Reply:
Since we're dealing here with runtime status, I think it makes more sense to put this method on the `Extralite` module and call it: `Extralite.runtime_status`.

Reply:
done